docs/website/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@sanomalearning/slds-docs",
```

Comment:
This should probably also have `"private": true`? Because the module won't be published?

Reply:
right